rialto_airflow/dags/harvest.py

```diff
@@ -169,20 +167,29 @@ def publish_openaccess(snapshot):
 
     @task()
     def upload_publish_files(snapshot):
-        logging.info(
-            f"Uploading {snapshot.path / 'publications.csv'} to file id {Variable.get('open_access_publications_file_id')}"
+        open_access_publications_filename = "publications.csv"
+        open_access_contributions_filename = "contributions.csv"
+        open_access_publications_file_id = google.get_file_id(
+            google.open_access_dashboard_folder_id(), open_access_publications_filename
+        )
+        open_access_contributions_file_id = google.get_file_id(
+            google.open_access_dashboard_folder_id(), open_access_contributions_filename
         )
```

Comment:
lookup the file ids of the files to replace by name instead of using configuration

Reply:
I have a slight preference of using one folder rather than different folders for each dashboard, as I expect there might be some reuse amongst visualizations. But I can live with it being this way.